var/spack/repos/builtin/packages/fastmath/package.py

```diff
@@ -0,0 +1,58 @@
+##############################################################################
```

Comment:
You should use #3133.  It needs to get merged, using it in this PR will be an incentive.